README.md

```diff
@@ -34,10 +39,109 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
 
 Here's the quickest way to get going with Rook.
 
-### Linux
+### Kubernetes
+
+This example shows how to build a simple, multi-tier web application on Kubernetes using persistent volumes enabled by Rook.
+
+#### Prerequisites
+
+This example requires a running Kubernetes cluster. To make sure you have a Kubernetes cluster that is ready for `rook`, you can [follow these quick instructions](demo/kubernetes/README.md).
+
```

Comment:
Would be good to highlight that we are looking for a way to make this work out of the box on K8S (and possibly creating a tracking issue in the kubernetes repo).